sdks/go/pkg/beam/runners/prism/internal/engine/elementmanager.go

```diff
@@ -688,6 +697,7 @@ func reElementResiduals(residuals []Residual, inputInfo PColInfo, rb RunBundle)
 					pane:      pn,
 					elmBytes:  elmBytes,
 					keyBytes:  keyBytes,
+					sequence:  len(unprocessedElements),
```

Comment:
Non-blocking. I know that len has an `O(1)` but just curious why not just `for i, w := range ws { sequence: i }`?

Reply:
Because that would be sequencing per window, (degenerate case, all 0) rather than by the whole residual set.